packages/connect-conformance/src/node/server.ts

```diff
@@ -72,14 +75,24 @@ export function run() {
         requestCert: true,
         rejectUnauthorized: true,
         ca: Buffer.from(req.clientTlsCert),
+        highWaterMark:
+          req.messageReceiveLimit === 0 ? undefined : req.messageReceiveLimit,
       };
     }
   }
   switch (req.httpVersion) {
     case HTTPVersion.HTTP_VERSION_1:
       server = req.useTls
         ? https.createServer(serverOptions, adapter)
-        : http.createServer(adapter);
+        : http.createServer(
+            {
+              highWaterMark:
+                req.messageReceiveLimit === 0
+                  ? undefined
+                  : req.messageReceiveLimit,
```

Comment:
This doesn't seem like a good idea. Can we simply disable the tests until we solve the underlying problem?

Reply:
Pushed by mistake, removing